rootfs/network/parse-config

```diff
@@ -103,9 +103,18 @@ EOF
     else
         echo 'iface eth0 inet dhcp'
         echo -n '    udhcpc_opts -t 10 -O search'
+        if HOSTNAME="$(parse_config HOSTNAME)"; then
+            # Send hostname to DHCP server
+            echo -n " -x hostname:$HOSTNAME"
+        else
+            # Ask for hostname from DHCP server
+            echo -n " -O hostname"
```

Comment:
Be consistent about use of `'` vs `"` here: if no substitution is required, use `'`, as elsewhere in this script.